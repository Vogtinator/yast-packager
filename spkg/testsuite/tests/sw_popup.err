2001-03-30 10:26:58 <3> sturm(30777) [Interpreter] clients/sw_single.ycp:312 Can't read YCP file 'dummyfile': Datei oder Verzeichnis nicht gefunden
