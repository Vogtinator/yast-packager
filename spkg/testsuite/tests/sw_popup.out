Dump	TEST 1: argument .testp -> show popup Versions ...
Read	.probe.architecture "i386"
Read	.target.root "/"
Read	.run.df [$["dummy":"on", "free":"Available", "name":"Mounted", "prz":"Capacity", "spec":"Filesystem", "used":"Used", "whole":"1024-blocks"], $["free":"144988", "name":"/", "prz":"93%", "spec":"/dev/sda1", "used":"1733600", "whole":"1981000"], $["free":"2124147", "name":"/usr", "prz":"66%", "spec":"/dev/sda3", "used":"4080331", "whole":"6543449"]]
SW_SINGLE: partition input: [$["free":144988, "name":"/", "used":1733600], $["free":2124147, "name":"usr", "used":4080331]]
Default value: no key 'ycp' in $["root":"/"]
Read	.target.ycp "/var/lib/YaST2/installmap.ycp" $[]
Default value: no key 'umount' in $["bash":0]
Execute	.target.umount "/var/adm/mount" 0
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 1
Read	.yast2.instsource.cdrelease 1234
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 1
Read	.yast2.instsource.cdrelease 1234
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/umount /var/adm/mount" 0
SW_SINGLE: Installed version: 7.1
Default value: no key 'umount' in $["bash":0]
Execute	.target.umount "/var/adm/mount" 0
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Return	`next
Dump	TEST 2: wrong filename given -> Error reading configuration file
Read	.probe.architecture "i386"
Read	.target.root "/"
Read	.run.df [$["dummy":"on", "free":"Available", "name":"Mounted", "prz":"Capacity", "spec":"Filesystem", "used":"Used", "whole":"1024-blocks"], $["free":"144988", "name":"/", "prz":"93%", "spec":"/dev/sda1", "used":"1733600", "whole":"1981000"], $["free":"2124147", "name":"/usr", "prz":"66%", "spec":"/dev/sda3", "used":"4080331", "whole":"6543449"]]
SW_SINGLE: partition input: [$["free":144988, "name":"/", "used":1733600], $["free":2124147, "name":"usr", "used":4080331]]
SW_SINGLE: POSTFIX: file
Default value: no key 'ycp' in $["root":"/"]
Read	.target.ycp "/var/lib/YaST2/installmap.ycp" $[]
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 1
Read	.yast2.instsource.cdrelease 1234
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 1
Read	.yast2.instsource.cdrelease 1234
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "test -f dummyfile" 0
Return	`cancel
Dump	TEST 3: wrong rpm-filename given -> File not found
Read	.probe.architecture "i386"
Read	.target.root "/"
Read	.run.df [$["dummy":"on", "free":"Available", "name":"Mounted", "prz":"Capacity", "spec":"Filesystem", "used":"Used", "whole":"1024-blocks"], $["free":"144988", "name":"/", "prz":"93%", "spec":"/dev/sda1", "used":"1733600", "whole":"1981000"], $["free":"2124147", "name":"/usr", "prz":"66%", "spec":"/dev/sda3", "used":"4080331", "whole":"6543449"]]
SW_SINGLE: partition input: [$["free":144988, "name":"/", "used":1733600], $["free":2124147, "name":"usr", "used":4080331]]
SW_SINGLE: POSTFIX: .rpm
Execute	.target.bash "test -f dummyfile.rpm" 1
Execute	.target.bash "/bin/umount /var/adm/mount" 1
Return	`next
Dump	TEST 4: CD number is set to 2 -> popup Please insert CD 2
Read	.probe.architecture "i386"
Read	.target.root "/"
Read	.run.df [$["dummy":"on", "free":"Available", "name":"Mounted", "prz":"Capacity", "spec":"Filesystem", "used":"Used", "whole":"1024-blocks"], $["free":"144988", "name":"/", "prz":"93%", "spec":"/dev/sda1", "used":"1733600", "whole":"1981000"], $["free":"2124147", "name":"/usr", "prz":"66%", "spec":"/dev/sda3", "used":"4080331", "whole":"6543449"]]
SW_SINGLE: partition input: [$["free":144988, "name":"/", "used":1733600], $["free":2124147, "name":"usr", "used":4080331]]
Default value: no key 'ycp' in $["root":"/"]
Read	.target.ycp "/var/lib/YaST2/installmap.ycp" $[]
Default value: no key 'umount' in $["bash":0]
Execute	.target.umount "/var/adm/mount" 0
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 2
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Execute	.target.bash "/bin/mount /dev/sr0 /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 2
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Return	`next
Dump	TEST 5: CD number is set to 2 and argument beginner -> show popup No access to ...
Read	.probe.architecture "i386"
Read	.target.root "/"
Read	.run.df [$["dummy":"on", "free":"Available", "name":"Mounted", "prz":"Capacity", "spec":"Filesystem", "used":"Used", "whole":"1024-blocks"], $["free":"144988", "name":"/", "prz":"93%", "spec":"/dev/sda1", "used":"1733600", "whole":"1981000"], $["free":"2124147", "name":"/usr", "prz":"66%", "spec":"/dev/sda3", "used":"4080331", "whole":"6543449"]]
SW_SINGLE: partition input: [$["free":144988, "name":"/", "used":1733600], $["free":2124147, "name":"usr", "used":4080331]]
SW_SINGLE: POSTFIX: nner
Default value: no key 'ycp' in $["root":"/"]
Read	.target.ycp "/var/lib/YaST2/installmap.ycp" $[]
Read	.probe.cdrom [$["dev_name":"/dev/sr0"]]
Execute	.target.bash "/bin/mount /dev/cdrom /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 2
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Execute	.target.bash "/bin/mount /dev/sr0 /var/adm/mount 2> /var/log/y2logMount" 0
Read	.yast2.instsource.cdnum 2
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Default value: no key 'umount' in $["bash":0]
Execute	.target.umount "/var/adm/mount" 0
Execute	.target.bash "/bin/umount /var/adm/mount" 0
Return	`next
